app/assets/stylesheets/common/components/user-card.scss

```diff
@@ -112,232 +126,606 @@
-      height: var(--avatar-width);
+    .names__primary,
+    .names__secondary {
+      a {
+        color: var(--primary);
+      }
     }
-    .new-user a {
-      color: var(--primary-low-mid);
+    p {
+      margin: 0 0 5px 0;
     }
   }
-  // user bio - suspension reason
+
+  // group bio
   .second-row {
     max-height: 150px;
     overflow: auto;
     .bio {
       a:not(.mention) {
         color: var(--tertiary);
       }
+      img {
+        max-width: 100%;
+        height: auto;
+      }
       .overflow {
         max-height: 60px;
         overflow: hidden;
       }
     }
-    .suspended {
-      color: var(--danger);
-      .suspension-reason-title,
-      .suspension-date {
-        font-weight: bold;
-      }
-    }
-    .profile-hidden,
-    .inactive-user {
-      font-size: var(--font-up-1);
-      margin-top: 0.5em;
-    }
   }
-  // featured topic
-  .featured-topic {
-    .desc {
-      color: var(--primary-high);
+}
+
+// styles for user cards only
+.d-user-card {
+  all: initial;
+  font-family: var(--font-family);
+  width: 480px;
+  border-radius: var(--d-border-radius);
+  box-shadow: 0px 4px 12px 0px rgba(0, 0, 0, 0.1);
+  background: var(--user-background);
+  background-color: var(--dominant-color);
+  overflow: hidden;
+
+  &__container {
+    padding: 5px;
+    backdrop-filter: var(--usercard-backdrop);
+    -webkit-backdrop-filter: var(--usercard-backdrop);
+  }
+  &__header {
+    position: relative;
+    height: 100px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: space-between;
+  }
+  &__badges {
+    display: flex;
+    border-radius: var(--d-border-radius);
+    gap: 0.5em;
+    margin-left: auto;
+  }
+  &__badges .user-badge {
+    background-color: transparent;
+    border: none;
+    margin-top: 0;
+    padding: 0;
+  }
+  &__badges .user-card-badge-link {
+    display: flex;
+    padding: 5px 6px;
+    background-color: rgba(var(--secondary-rgb), 1);
+    border-radius: var(--d-border-radius);
+  }
+  &__badges .user-card-badge-link:hover {
+    background-color: var(--secondary-very-high);
+  }
+  &__badges .badge-display-name {
+    display: none;
+  }
+  &__badges .svg-icon {
+    width: 1.125em;
+    height: 1.125em;
+  }
+  &__badges-more {
+    font-size: var(--font-down-1);
+    display: flex;
+    padding: 5px 6px;
+    background-color: rgba(var(--secondary-rgb), 1);
+    border-radius: var(--d-border-radius);
+    align-items: center;
+  }
+  &__badges-more:hover {
+    background-color: var(--secondary-very-high);
+  }
+  &__badges-more a {
+    font-size: var;
+    color: var(--primary);
+  }
+  &__edit-btn {
+    cursor: pointer;
+    position: absolute;
+    right: 0;
+    bottom: 5px;
+    border: none;
+    font-size: 14px;
+    font-weight: normal;
+    transition: all 0.25s;
+    opacity: 50%;
+    color: var(--primary) !important;
+    background-color: rgba(var(--secondary-rgb), 0.75) !important;
```

Comment:
The `!important`s here feel a bit heavy-handed. Maybe a theme might want to override these?